src/inference/include/openvino/runtime/auto/properties.hpp

```diff
@@ -28,5 +28,50 @@ static constexpr Property<bool> enable_startup_fallback{"ENABLE_STARTUP_FALLBACK
  * selected device
  */
 static constexpr Property<bool> enable_runtime_fallback{"ENABLE_RUNTIME_FALLBACK"};
+
+/**
+ * @brief Enum to define schedult policy of inference request for MULTI
+ * @ingroup ov_runtime_cpp_prop_api
+ */
+enum class SchedulePolicy {
+    ROUND_ROBIN = 0,
+    DEVICE_PRIORITY = 1,
```

Comment:
please, add documentation for each value

Reply:
Updated.